app/src/main/java/com/dscvit/handly/network/ApiInterface.kt

```diff
@@ -0,0 +1,31 @@
+package com.dscvit.handly.network
+
+import com.dscvit.handly.model.auth.AuthResponse
+import com.dscvit.handly.model.auth.SigninRequest
+import com.dscvit.handly.model.auth.SignupRequest
+import com.dscvit.handly.model.collection.*
+import retrofit2.Response
+import retrofit2.http.*
+
+interface ApiInterface {
+
+    @POST("user/create/")
```

Comment:
Do we need the trailing `/`? (Is it django?)

Reply:
Yes saar django it is